packages/rich-text/src/helpers/editor.ts

```diff
@@ -231,3 +233,17 @@ export const isInlineOrText = (node: Node) => {
   // either text or inline elements
   return Text.isText(node) || (Element.isElement(node) && INLINE_TYPES.includes(node.type));
 };
+
+export const focus = (editor: PlateEditor) => {
+  const x = window.scrollX;
+  const y = window.scrollY;
+
+  ReactEditor.focus(editor);
+
+  // Safari has issues with `editor.focus({ preventScroll: true })`, it ignores the option `preventScroll`
+  if (IS_SAFARI) {
```

Comment:
I don't have access to Safari so I'm curious how it works in a real browser? does the browser scrolls to the top and then get forced (by the code above) to get back to the correct location? wouldn't that be more jarring UX-wise?

Reply:
You can't really see the transition but there are a few tweaks when using table actions, in this case, yes, you can see some transition happening, but that was the only case, everything else works as fine as in other browsers.